docs/networks/mainnet.md

```diff
@@ -0,0 +1,104 @@
+# Mainnet Configuration
+
+The Graph Network's testnet is on Ethereum, and network information can be found at https://thegraph.com/explorer/network.
```

Comment:
Probably shouldn't mention "testnet" here in the Mainnet doc.  

I'd suggest simplifying this line to: 
"Network information can be found at https://thegraph.com/explorer/network"

Reply:
good catch!